addon/components/modal-dialog.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import template from '../templates/components/modal-dialog';
+
+var computed = Ember.computed;
+
+export default Ember.Component.extend({
+  tagName: '', // positioned-container is the container you're looking for
```

Comment:
Ah, I'll update this. The modal dialog component is itself tagless, which may confuse people perusing the source in search of what renders *the* div. The actual div comes from the positioned container component.